client/commonFramework/update-preview.js

```diff
@@ -13,7 +13,10 @@ window.common = (function(global) {
   window.__err = null;
   window.loopProtect.hit = function(line) {
     window.__err = new Error(
-      'Potential infinite loop at line ' + line
+      "Potential infinite loop at line " + line + ".\n\
+      To disable loop protection: add \"// noprotect\" to your code.\n\
```

Comment:
Try putting each part of this string on its own line. The newlines are not needed.



```js
 "Potential infinite loop at line " +
 line +
 ". To disable loop protection: add \"// noprotect\" to your code."
```
The newline characters are not needed.

 Also, please enable linting in your editor as this is not syntacticly correct.